development/test-log-collector/README.md

```diff
@@ -0,0 +1,62 @@
+# Test Log Collector
+
+## Overview
+
+The purpose of the Test Log Collector is to gather logs from the testing pods and to send them to the appropriate slack channels.
+It is intended to be run after the Octopus ClusterTestSuite, as it uses pods labels from Octopus. Test Log Collector will send logs only from the newest, finalised ClusterTestSuite.
+
+## Requirements
+
+In order to send the message to any Slack channel you need to have [Slack app](https://api.slack.com/start) added to that channel, and you need to have its token. Slack app tokens typically have a `xoxb-` prefix. Slack app is required to have following bot token scopes:
+
+- channels:history
+- chat:write
+- files:write
+
+## Usage
+
+To use the test-log-collector, navigate to its chart directory and run it with appropriate parameters. See the example:
+
+```bash
+helm install test-log-collector \
+--namespace kyma-system ./chart/test-log-collector \
+--set slackToken=${slack_token} \
+--wait \
+--timeout 600s
+```
+
+## Configuration
+
+Test Log Collector dispatches logs to particular Slack channels based on configuration file, located [here](./chart/test-log-collector/files/config.yaml). This config file is a list, which uses following fields to configure the application:
+
+| Parameter             | Description                                                                                                                                                                            |
+| --------------------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| **testCases**         | List of test names, from which logs should be sent to particular channel. Specifying `default` item in that list creates a sink for test cases, which have unspecified target channel. |
+| **channelName**       | Name of the channel, to which logs are sent. **Must** start with "#"                                                                                                                   |
+| **channelID**         | ID of the channel, to which logs are sent                                                                                                                                              |
+| **onlyReportFailure** | Indicates whether only logs from failed tests should be sent to Slack channel                                                                                                          |
+
+You can get available test cases by running `kyma test definitions`.
+
+`channelID` can be obtained by right clicking channel in Slack and choosing `Copy link` option. Channel ID is the last part of that link. If the channel link is `https://example.slack.com/archives/CPBNQ4KNG` then the ID equals `CPBNQ4KNG`.
```

Comment:
```suggestion
You can obtain `channelID` by right-clicking the channel on Slack and choosing the **Copy link** option. `channelID` is the last part of that link. If the channel link is `https://example.slack.com/archives/CPBNQ4KNG`, the ID equals `CPBNQ4KNG`.
```

Reply:
I changed the passive voice into the active voice which we rather prefer using :)